range.ts

```diff
@@ -0,0 +1,12 @@
+function validRange (range: any) {
+    return range.min <= range.middle && range.middle <= range.max;
+}
+
+const range = {
+    min: 5,
+    middle: 10,    // TSLint will warn about unsorted keys here
+    max: 20
+};
+
+console.log(validRange(range));
```

Comment:
__[tslint]__
* Calls to 'console.log' are not allowed.

<!-- Comment by [SideCI](https://sideci.com) -->